### DISCUSSION TITLE
Logitech g923(xbox) pedal order

### DISCUSSION THREAD

**Discussion by jeisom** [berarma/oversteer#156] [Q&A]
I have a g923 and the pedals from L to R are showing as brake, accel and clutch in oversteer.  Is there anyway to change the order to clutch, brake , accel ?

**Comment by berarma** [comment]
I was just remapping pedals for other wheels and did this one too. Please, can you test the latest master commit?

**Reply by jeisom** [reply]
Ok, now the gas pedal does the brakes, the brakes do the clutch and the clutch is the accel.

I was testing originally the 0.7.2 release btw.

**Comment by berarma** [comment]
I didn't understand it correctly the first time. It should be correct now.

**Reply by jeisom** [reply]
Thanks, that worked.
